test/e2e/manifest/manifest.go

```diff
@@ -105,42 +92,47 @@ func StatefulSetFromManifest(fileName, ns string) (*appsv1.StatefulSet, error) {
 	return &ss, nil
 }
 
-// DaemonSetFromManifest returns a DaemonSet from a manifest stored in fileName in the Namespace indicated by ns.
-func DaemonSetFromManifest(fileName, ns string) (*appsv1.DaemonSet, error) {
-	var ds appsv1.DaemonSet
-	data, err := testfiles.Read(fileName)
-	if err != nil {
-		return nil, err
+// DaemonSetFromManifest reads a .json/yaml file and returns the daemonset in it.
+func DaemonSetFromManifest(url string) (*appsv1.DaemonSet, error) {
```

Comment:
We should update the name and description of this function to indicate that the manifest is downloaded from a URL.
`<Something>FromManifest` makes me think that we are passing the json or yaml directly instead of specifying a URL.